lib/assets/javascripts/twine.coffee

```diff
@@ -462,7 +457,7 @@
 
   setupEventBinding = (eventName) ->
     Twine.bindingTypes["bind-event-#{eventName}"] = (node, context, definition) ->
-      onEventHandler = (event, data) ->
+      onEventHandler = (event, data = event.detail) ->
```

Comment:
Is this something that jQuery does?

Reply:
I think I found it in https://api.jquery.com/on/
![Screen Shot 2021-02-22 at 10 52 57 AM](https://user-images.githubusercontent.com/150352/108733073-24a05500-74fc-11eb-8279-8e0f10370125.png)

We use this in admin via `$someEl.trigger('click', someObj)`

Reply:
Yep, that's it @qq99. I originally removed the data functionality but @mattdrose had the idea to make use of the native [`detail` property](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/detail) of `CustomEvent`. This means that folks _could_ upgrade to something similar [as shown in this test](https://github.com/Shopify/twine/blob/5f28a75694c354aa9127868faf797026efa24523/test/twine_test.coffee#L355-L364).